backend/common/src/main/java/ai/verta/modeldb/common/authservice/UACApisUtil.java

```diff
@@ -42,10 +47,12 @@ public class UACApisUtil {
   private static final Logger LOGGER = LogManager.getLogger(UACApisUtil.class);
   protected final FutureExecutor executor;
   protected final UAC uac;
+  protected final boolean isPermissionV2;
```

Comment:
Why is this flag needed? I thought search by owner works for both permission v1 and v2?

Reply:
It is not in used so removed